src/server/routes/index.js

```diff
@@ -13,6 +13,10 @@ const jwtCheck = jwt({
 
 const router = express.Router();	// eslint-disable-line new-cap
 
+router.get('/', (req, res) => {
```

Comment:
I think /health-check should remain - being deliberate about an endpoint where the health check lives means less chance of bumping into it later on IMO. I feel like root should do * something * though...maybe a clear status 403 or a 304 redirect to the apiary docs?? thoughts?